### PR TITLE
Create LIT7063Devils.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
According to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2596.

### CODE REVIEW

**Review by nafisa-valieva — COMMENTED** [review]
Ok!

File: new/LIT7063Devils.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7063Devils" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አላሁማ፡ ወያኑራ፡ ሐሽም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾAlahuma wa-yānurā ḥašǝm</title>
```
Comment (karljonaskarlsson) [inline new/LIT7063Devils.xml:9]: Maybe it is better to transcribe "ʾAllāhumma" (cf. ٱللَّٰهُمَّ) – at least that is what I would search for if I was looking for this text.

**Review by karljonaskarlsson — APPROVED** [review]

File: new/LIT7063Devils.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                <title xml:lang="en" corresp="#t1">ʾAlahuma and Yānurā ḥašǝm</title>
```
Comment (eu-genia) [inline new/LIT7063Devils.xml:10]: I am not sure that this translation makes sense

At the same time, this is obviously a sequence of Arabic/Islamic names/terms used for asmat, which could be interesting if one looks for Muslim/Arabic references, so worth marking up

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT7063Devils.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7063Devils.xml]: What is your suggestion? Do you want another translation or should I not translate it and delete the translation in this record?

Comment (eu-genia) [reply new/LIT7063Devils.xml]: I would delete the translation and rather add some keyword/description to make it searchable if one looks for Islamic allusions, and Arabic loans; at the moment these aspects are obscured and not retrievable

Comment (eu-genia) [reply new/LIT7063Devils.xml]: possibly this or similar prayer is among those which were studied by Griaule, he published on Islamic formulae used in Christian prayers, could be checked (_Le livre de recettes d'un dabtara abyssin_). Also Strelcyn published separately on the phenomenon.

Comment (CarstenHoffmannMarburg) [reply new/LIT7063Devils.xml]: > possibly this or similar prayer is among those which were studied by Griaule, he published on Islamic formulae used in Christian prayers, could be checked (_Le livre de recettes d'un dabtara abyssin_). Also Strelcyn published separately on the phenomenon.

OK. I will check that as soon as possible.

Comment (eu-genia) [reply new/LIT7063Devils.xml]: not yet updated?

Comment (CarstenHoffmannMarburg) [reply new/LIT7063Devils.xml]: No, I missed to have a look to the HLCEE library, when I was in Hamburg. I will possibly find Griaule in Marburg next week or I will do it in two weeks, when I am back in Hamburg.

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Comment by eu-genia** [comment]
https://github.com/BetaMasaheft/Works/pull/1316

**Pull request merged**